Olfeo/olfeo-saas/_meta/manifest.yml

```
@@ -0,0 +1,8 @@
+uuid: ce4ab9ba-4ed7-420d-af30-f144229f37ee
+name: Olfeo SAAS
```

For new released integration, we add a flag for a limited-time:
```suggestion
name: Olfeo SAAS [BETA]
```